obspy/signal/konnoohmachismoothing.py

```diff
@@ -85,10 +77,13 @@ def konno_ohmachi_smoothing_window(frequencies, center_frequency,
         smoothing_window = np.zeros(len(frequencies), dtype=frequencies.dtype)
         smoothing_window[frequencies == 0.0] = 1.0
         return smoothing_window
-    # Calculate the bandwidth*log10(f/f_c)
-    smoothing_window = bandwidth * np.log10(frequencies / center_frequency)
-    # Just the Konno-Ohmachi formulae.
-    smoothing_window[...] = (np.sin(smoothing_window) / smoothing_window) ** 4
+    # Disable div by zero errors and return zero instead
+    with np.errstate(divide='ignore', invalid='ignore'):
+        # Calculate the bandwidth*log10(f/f_c)
+        smoothing_window = bandwidth * np.log10(frequencies / center_frequency)
+        # Just the Konno-Ohmachi formulae.
+        smoothing_window[...] =\
```

Comment:
http://stackoverflow.com/questions/118370/how-do-you-use-the-ellipsis-slicing-syntax-in-python

Don't think it is necessary here. This is, anyway, a piece of code from @krischer.